### DISCUSSION TITLE
How to set culture for date format - running

### DISCUSSION THREAD

**Discussion by RobSmyth** [bagetter/BaGetter#169] [General]
I'm running BaGetter on an Ubuntu box and I'm seeing date format of mm/dd/yyyy. Does BaGetter have a way of setting current culture?

![image](https://github.com/user-attachments/assets/111ebf7c-2c8a-472a-abd0-c52cae7ff9a9)

Those date are not Feb 2, 2024 :-)  I'm seeking dd/mm/yyyy but more global format like yyyy-mm-dd would do.

I've changed the Ubuntu locale and rebooted no effect. I found the `ToShortDateString` call in the code but I have not found a place that sets the culture nor familiar with dotnet culture on Linux nor Docker. I wonder if you have an answer before I search further.

**Comment by Regenhardt** [comment]
We're tracking this in https://github.com/bagetter/BaGetter/issues/58 and have tried to fix it, which didn't work the first try. When my bachelor's thesis is finished I'll look into it some more, until then I would absolutely accept a PR to fix it though.
